enterprise/cmd/worker/internal/codeintel/indexing/iface.go

```diff
@@ -27,6 +27,20 @@ type DBStoreShim struct {
 	*dbstore.Store
 }
 
+// TODO: I don't like this name very much
+type IndexingSettingStore interface {
+	GetLastestSchemaSettings(context.Context, api.SettingsSubject) (*schema.Settings, error)
+}
+
+// settingStore <- variable and the type
+// setting_store = SettingStore()
+
+// TODO: I also don't like this name very much :)
+// Can probably just re-use the names and get used to that.
```

Comment:
```suggestion
type IndexingSettingStore interface {
	GetLastestSchemaSettings(context.Context, api.SettingsSubject) (*schema.Settings, error)
}

```

I think these names are good enough for the first pass :)

It's just ifaces for tests so it's not any huge deal since the places they're used are rather local.